third_party/java/java_tools/update_java_tools.sh

```diff
@@ -43,18 +48,30 @@ fi
 
 # Stores the names of the tools required for update.
 tools_to_update=()
+
+# Stores the workspace relative path of all the tools that were updated
+# (e.g. third_party/java/java_tools/JavaBuilder_deploy.jar)
+updated_tools=()
+
 if [[ ! -z "$@" ]]
 then
    # Update only the tools specified on the command line.
    tools_to_update=("$@")
+   echo "The sources archive was NOT created. Please create it manually!"
 else
   # If no tools were specified update all of them.
   tools_to_update=(${!tool_name_to_target[*]})
+  # Create the sources archive only when all the tools were updated.
+  zip -r third_party/java/java_tools/java_tools-srcs.zip src/java_tools/buildjar/* \
```

Comment:
This is non-deterministic, can you please try to remove the timestamps, so it does give the same md5 sum every time?

Reply:
Good point. I added `-X` option which doesn't include file times